bundles/mapping/mapmodule/domain/AbstractLayer.js

```diff
@@ -1017,6 +1017,26 @@ Oskari.clazz.define(
         getOptions: function () {
             return this._options;
         },
+        /**
+         * @method getAttributions
+         * @return {Array<String>} layer attributions
+         */
+        getAttributions: function () {
+            var attributions = this._options.attributions;
+            if (!attributions) {
+                return;
+            }
+            if (!Array.isArray(attributions)) {
+                attributions = [attributions];
+            }
+            return attributions.map(obj => {
+                if (typeof obj === 'string') {
+                    return obj;
+                }
+                const { label, link } = obj;
+                return link ? `<a href="${link}">${label}</a>` : label;
```

Comment:
Move UI specific formatting out of AbstractLayer to VectorTileLayerPlugin.js or even AbstractMapLayerPlugin.js